Libraries/LibWeb/SVG/SVGSVGElement.cpp

```diff
@@ -309,9 +314,14 @@ SVGSVGElement::NaturalMetrics SVGSVGElement::negotiate_natural_metrics(SVG::SVGS
             return {};
         }
 
-        // FIXME: 2. If an SVG View is active:
-        // FIXME:    1. let viewbox be the viewbox defined by the active SVG View
-        // FIXME:    2. return viewbox.width / viewbox.height
+        // 2. If an SVG View is active:
+        if (auto active_view_element = svg_root.active_view_element(); active_view_element && active_view_element->view_box().has_value()) {
+            // 1. let viewbox be the viewbox defined by the active SVG View
+            auto view_box = active_view_element->view_box().value();
+
+            // 2. return viewbox.width / viewbox.height
+            return view_box.width / view_box.height;
```

Comment:
nit: it'd probably advisable to take some of the logic down below to avoid possible division by zero.